spec/lib/hyrax/hirmeos/metrics_tracker_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+require 'hyrax/hirmeos/metrics_tracker'
+require 'rails_helper'
+
+RSpec.describe Hyrax::Hirmeos::MetricsTracker do
+  subject(:tracker) { described_class.new }
+  let(:work) { create(:work) }
+
+  before do
+    Rails.application.routes.default_url_options[:host] = 'localhost:3000'
+    tracker.username = "UsernameTest"
+    tracker.password = "Password"
+    tracker.metrics_base_url = "https://dummy-metrics-api/events"
+    tracker.token_base_url = "https://tokens.dummy.endpoint/"
+    tracker.translation_base_url = "https://translator.dummy.endpoint"
```

Comment:
I'm surprised this works since the properties are class attributes and should be set like in the initializer:
```suggestion
    Hyrax::Hirmeos::MetricsTracker.username = "UsernameTest"
    Hyrax::Hirmeos::MetricsTracker.password = "Password"
    Hyrax::Hirmeos::MetricsTracker.metrics_base_url = "https://dummy-metrics-api/events"
    Hyrax::Hirmeos::MetricsTracker.token_base_url = "https://tokens.dummy.endpoint/"
    Hyrax::Hirmeos::MetricsTracker.translation_base_url = "https://translator.dummy.endpoint"
```